src/components/pages/HomePage/Index.css.tsx

```diff
@@ -0,0 +1,52 @@
+import { css } from '@emotion/react';
+
+const exampleChat = css`
+  position: relative;
+  height: 50vh;
+  width: auto;
+
+  @media (max-width: 500px) {
+    height: 40vh;
+  }
+`;
+
+const exampleOverview = css`
+  position: relative;
+  height: 75vh;
+  width: auto;
+
+  @media (max-width: 500px) {
+    height: 30vh;
+  }
+`;
+
+const finalTextBox = css`
+  width: 500px;
+  margin: auto;
+
+  @media (max-width: 500px) {
+    width: 100%;
+    min-width: unset;
+  }
+`;
```

Comment:
Using max-width removes the need for a media query
```suggestion
const finalTextBox = css`
  max-width: 500px;
  margin: auto;
`;
```